sources/org.osbuild.ostree

```diff
@@ -49,6 +60,19 @@ SCHEMA = """
                   "description": "GPG keys to verify the commits"
                 }
               },
+              "secrets": {
```

Comment:
Already exists in the properties, see just below.

Reply:
Oh of course I was so focused to get indentation correctly...